### ISSUE TITLE
Error when transcoding with current FFmpeg, fixed in new ones

### ISSUE THREAD

**Issue by SubJunk** [UniversalMediaServer/UniversalMediaServer#1886]
When using a command like:

```
ffmpeg -y -loglevel info -i video.mkv -filter_complex [0:v][0:s:0]overlay,pad=iw:iw/(16/9):0:(oh-ih)/2,scale=1920:1080 -bufsize 31250k -maxrate 31250k -crf 19 -ac 6 -ab 640k -c:a ac3 -c:v libx264 -tune zerolatency -level 31 -pix_fmt yuv420p -f mpegts -preset superfast file.ts
```

I get the error:

```
Unable to parse option value "(null)" as sample format
```

and the transcoding fails.
I have confirmed it is fixed with newer FFmpeg versions, with no changes to the commands, so we should try to update to a newer version.

**Comment by valib** [comment]
@SubJunk have you tried to use example from  https://trac.ffmpeg.org/wiki/HowToBurnSubtitlesIntoVideo especially `"[0:v][0:s]overlay[v]" -map "[v]" -map 0:a` ?

**Comment by SubJunk** [comment]
@valib I have referenced that article before but not this time

**Comment by SubJunk** [comment]
Fixed in #1951
